far/farlang.templ.m4

```diff
@@ -4527,7 +4527,7 @@ MEditAutoDetectCodePage
 
 MEditShareWrite
 "Разрешить редактирование открытых для записи &файлов"
-"Allow editing files ope&ned for writing"
+"Allow editing files opened for &writing"
```

Comment:
Reassigning hotkeys is not really the point of this PR, is it?

It can also break someone's muscle memory or macros or whatever, are you sure you want to change that?

Reply:
What I'm saying is that the point of this PR is introducing a new feature.
The smaller the PR and the fewer other features it affects, the faster it will get merged.

You can, for example, add `[ ] Show line numbers` without any hotkey at all, avoiding any conflicts with other hotkeys and any need to touch or discuss them now, and then raise a follow-up PR with hotkeys rearrangements only.

Or you can touch this highly subjective topic here.
The changes you suggest might be totally reasonable - indeed, maximum possible association with the item name is a logical goal, and personally I don't mind the new assignments at all, but other people might have different opinions (and because of the bikeshedding they absolutely will), so we might spend unholy amounts of time in this rabbit hole.